### PR TITLE
Trending tags a.k.a. hot topics

### PR DESCRIPTION
**Pull request by leogilmour**
This has no limit / suggestion currently.

### CODE REVIEW

**Comment by leogilmour** [comment]
[WIDGET: Trending Tags](https://trello.com/c/rQyFBFOb/225-widget-trending-tags)

File: widget-loader-acf.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		array (
+		'key' => $key . '_basic_details_tab',
+		'label' => 'Basic Details',
+		'type' => 'tab',
+		'placement' => $widgetplacement,
+		),
+		array (
+			'key' => $key . '_tags_manual',
+				'label' => 'Tags',
+				'name' => 'tags',
```
Comment (ecoad) [inline widget-loader-acf.php]: The `name` should match the the `key` minus the prefix-y bit (`$key`)

So if your `key` is `widget_trending_tags_tags_manual` then the `name` should be `tags_manual`.

File: widget-loader-acf.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	'sub_fields' => array (
+		array (
+		'key' => $key . '_basic_details_tab',
+		'label' => 'Basic Details',
+		'type' => 'tab',
+		'placement' => $widgetplacement,
+		),
+		array (
+			'key' => $key . '_tags_manual',
+				'label' => 'Tags',
```
Comment (ecoad) [inline widget-loader-acf.php]: Whitespace issue here

File: widget-loader-acf.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+				'name' => 'tags',
+				'type' => 'taxonomy',
+				'taxonomy' => 'post_tag',
+				'field_type' => 'multi_select',
+				'add_term' => 0,
+				'save_terms' => 0,
+				'return_format' => 'object'
+		),
+		array (
+		'key' => $key . '_advanced_details_tab',
```
Comment (ecoad) [inline widget-loader-acf.php]: Whitespace: Can you indent the code inside the array please.

**Review by ecoad — CHANGES_REQUESTED** [review]
Minor tweaks todo - thanks

**Review by ecoad — APPROVED** [review]
Thanks

**Pull request merged**